packages/@dcl/inspector/src/lib/babylon/decentraland/EcsEntity.ts

```diff
@@ -97,6 +97,66 @@ export class EcsEntity extends BABYLON.TransformNode {
     return new BABYLON.BoundingInfo(min, max)
   }
 
+  getTransformedVertices(mesh: BABYLON.AbstractMesh) {
+    const positions = mesh.getVerticesData(BABYLON.VertexBuffer.PositionKind)!
+
+    const worldVertices = []
+    const scaling = mesh.scaling
+    const rotationQuaternion = mesh.rotationQuaternion || BABYLON.Quaternion.Identity()
+    const position = mesh.position
+
+    for (let i = 0; i < positions.length; i += 3) {
+      const vertex = BABYLON.Vector3.FromArray(positions, i)
+
+      // Apply scaling
+      vertex.multiplyInPlace(scaling)
+
+      // Apply rotation
+      vertex.rotateByQuaternionToRef(rotationQuaternion, vertex)
+
+      // Apply translation
+      vertex.addInPlace(position)
+
+      worldVertices.push(vertex)
+    }
+
+    return worldVertices
+  }
+
+  computeMeshBoundingBox(mesh: BABYLON.AbstractMesh) {
+    const worldVertices = this.getTransformedVertices(mesh)
+
+    let min = new BABYLON.Vector3(Number.MAX_VALUE, Number.MAX_VALUE, Number.MAX_VALUE)
+    let max = new BABYLON.Vector3(-Number.MAX_VALUE, -Number.MAX_VALUE, -Number.MAX_VALUE)
+
+    for (const vertex of worldVertices) {
+      min = BABYLON.Vector3.Minimize(min, vertex)
+      max = BABYLON.Vector3.Maximize(max, vertex)
+    }
+
+    return { min, max }
+  }
+
+  getGroupMeshesBoundingBox() {
+    const children = this.getChildMeshes(false)
+    if (children.length === 0) {
+      return null // No children to calculate the bounding box
+    }
+
+    let groupMin = new BABYLON.Vector3(Number.MAX_VALUE, Number.MAX_VALUE, Number.MAX_VALUE)
+    let groupMax = new BABYLON.Vector3(-Number.MAX_VALUE, -Number.MAX_VALUE, -Number.MAX_VALUE)
```

Comment:
are we sure we want to use `Number.MAX_VALUE` instead of `Number.MAX_SAFE_INTEGER` & `Number.MIN_SAFE_INTEGER`?

Reply:
good catch -- updating it!